src/main/java/marquez/db/JobDao.java

```diff
@@ -32,12 +32,12 @@
   JobVersionDao createJobVersionDao();
 
   @SqlUpdate(
-      "INSERT INTO jobs (guid, name, namespace_guid, description, input_dataset_urns, output_dataset_urns) "
-          + " VALUES (:guid, :name, :namespaceGuid, :description, :inputDatasetUrns, :outputDatasetUrns)")
-  public void insert(@BindBean Job job);
+      "INSERT INTO jobs (guid, name, namespace_guid, description, input_dataset_urns, output_dataset_urns, type) "
+          + " VALUES (:guid, :name, :namespaceGuid, :description, :inputDatasetUrns, :outputDatasetUrns, :type)")
+  void insert(@BindBean Job job);
 
   @SqlUpdate("UPDATE jobs SET current_version_guid = :currentVersionGuid WHERE guid = :jobGuid")
-  public void setCurrentVersionGuid(UUID jobGuid, UUID currentVersionGuid);
+  void setCurrentVersionGuid(UUID jobGuid, UUID currentVersionGuid);
```

Comment:
why did this change? does it have something to do with job types?

Reply:
Oh, I left a review comment down below about this but basically it's unnecessary and the IDE gives a warning saying so. Figured it would be nice clean up little things.